lib/person.rb

```diff
@@ -29,5 +30,9 @@ def to_s
     "Person (id: #{@id}, name: #{@name}, age: #{@age}, parent_permission: #{@parent_permission})"
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ]  @kifle23 Great work adding a method to add rental in the person class, however in order to achieve the has-many relationship the method needs to include book, date, and the person himself. (kindly check the suggested code in the book class)

Reply:
Thank you so much @danmainah